sda-commons-metadata-context/src/main/java/org/sdase/commons/spring/boot/metadata/context/MetadataContextHolder.java

```diff
@@ -7,23 +7,23 @@
  */
 package org.sdase.commons.spring.boot.metadata.context;
 
-class MetadataContextHolder {
+public class MetadataContextHolder {
```

Comment:
I would not make this class and it's methods public but use `MetadataContext` in the interceptor.